src/artm/core/token.h

```diff
@@ -10,46 +10,32 @@
 #include "boost/functional/hash.hpp"
 
 #include "artm/core/common.h"
-#include "artm/core/transaction_type.h"
 
 namespace artm {
 namespace core {
 
 typedef std::string ClassId;
-const std::string DefaultClass = "@default_class";
-const std::string DocumentsClass = "@documents_class";
+typedef std::string TransactionTypeName;
 
-// Token is a triple of keyword, its class_id (also known as tokens' modality) and type of the transaction.
+const ClassId DefaultClass = "@default_class";
+const ClassId DocumentsClass = "@documents_class";
+const TransactionTypeName DefaultTransactionTypeName = "@default_transaction";
```

Comment:
RE DefaultTransactionTypeName
I think it was a mistake to have ``@default_class`` as a label for default modality. This is quite long and not very useful. 
Is there anything that will go wrong is we set ``DefaultTransactionTypeName`` to an empty string? And eventually fix default class id to "" (and add upgrader to all batches so that @default_class gets changed to "").